Assets/Scripts/Audio/UISoundEffects.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class UISoundEffects : MonoBehaviour
+{
+	//THIS CLASS IS FOR EXECUTING THE SOUND OF THE UI
+
+	public void CoinPurseSound() 
+	{
+		FindObjectOfType<AudioManager>().PlaySound("Coin Purse");
```

Comment:
we should try to avoid scattering FindObjectOfType everywhere since it's slow. it'd be better to put it in globals. you can do that like this:

In Globals.cs:
````
    . . .
    public static QuestSystem Quests => Get<QuestSystem>();
    public static MiniGames MiniGames => Get<MiniGames>();
    // add yours here
    public static AudioManager Audio => Get<AudioManager>();
````

and then in AudioManager's Awake:
````
		Globals.Register(this);
````

Then you can access it anywhere in the code like this:
````
    Globals.Audio.PlaySound("Trade Swap");
````

i saw you had looked at making it a static singleton with an AudioManager.instance style. This Globals style is just another way of writing that same concept. It's called service locator officially. I prefer it over the usual static singleton manager way because it's easier to change things out in a centralized place vs every class having its own unique static instance floating around.